Source/Scene/Scene.js

```diff
@@ -1405,6 +1411,55 @@ define([
             }
         },
 
+        /**
+         * The value used for gamma correction.
+         * @memberof Scene.prototype
+         * @type {Number}
+         * @default 2.4
+         */
+        gamma : {
+            get : function() {
+                return this._context.uniformState.gamma;
+            },
+            set : function(value) {
+                this._context.uniformState.gamma = value;
+            }
+        },
+
+        /**
+         * Whether or not to use high dynamic range rendering.
+         * @memberof Scene.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        highDynamicRange : {
+            get : function() {
+                return this._hdr;
+            },
+            set : function(value) {
+                var context = this._context;
+                var hdr = value && context.depthTexture && (context.colorBufferFloat || context.colorBufferHalfFloat);
```

Comment:
Here, especially below, and throughout, these statements are easier to read when written as `var foo = (expression);`.

Could be a good one for the Coding Guide if it isn't already in there.